### DISCUSSION TITLE
GUI/graphical focus?

### DISCUSSION THREAD

**Discussion by DanHolli** [darlinghq/darling#1534] [General]
Is implementing graphical translation a goal of this project?
I couldn't seem to find any mention of goals or milestones.

I was wondering because there are a few apps I'd like to use that are only run-able on old Macs namely Sling's now defunct Sling Player
I understand app like that and games are a ways off, just wanting to know. thank you!

**Comment by CuriousTommy** [comment]
> Is implementing graphical translation a goal of this project?

Yes (all the foundation work needed to run a GUI is already implemented, Darling can run a simple hello world app. The only issue is that Darling doesn't support much beyond that).

However, there are other issues with Darling that I want to work on first before I focus on improving GUI support (Add ARM64 support, rework some of Darlings logic, improve 3rd party language/CLI apps support)

**Reply by DanHolli** [reply]
Thank you for the response

I just tested it out (finally) last night, and while it may seem basic to average users (mostly just terminal) I really appreciate your and the other's hard work and think it's really cool that it is in the state that it is

I wish to see this advance and hopefully it'll become as useful as Wine

For the maintainer, sorry there are/were people bringing unrelated politics into this
